modules/nuxt-github-api.yml

```diff
@@ -0,0 +1,14 @@
+name: nuxt-github-api
+description: Nuxt module for GitHub v4 API
+repo: lindsaykwardell/nuxt-github-api
+npm: nuxt-github-api
+icon: ''
```

Comment:
Would be nice to add the github.svg icon :)

Reply:
In fact best in accordance with the name of the module :) 
Should be great now 👍